pkg/repository/maintenance/maintenance.go

```diff
@@ -460,6 +460,13 @@ func buildJob(cli client.Client, ctx context.Context, repo *velerov1api.BackupRe
 		}
 	}
 
+	podAnnotations := map[string]string{}
+	for _, k := range util.ThirdPartyAnnotations {
+		if v := veleroutil.GetVeleroServerAnnotationValue(deployment, k); v != "" {
+			podAnnotations[k] = v
+		}
+	}
```

Comment:
The current list of "well known annotations" only include aws.
https://github.com/vmware-tanzu/velero/blob/883e3e4aae976a5b6bcbf2c9289146625269e27a/pkg/util/third_party.go#L23-L25

Do we need to add more to really address https://github.com/vmware-tanzu/velero/issues/8811?

Reply:
We don't need it as far as I understand.
The problem happens just because the maintenance pod has inherited some labels/annotations from Velero server pod. While in a previous PR, we have changed the code not to inherit any labels/annotations from it, so the problem should have been fixed.

Reply:
This PR is to allow users to add third party annotations, which solves problem like AWS IAM authentication.

Reply:
For other third party labels/annotations, if users want to request more besides the existing ones in `ThirdPartyAnnotations` or `ThirdPartyLabels`, they should submit issues & separate PRs.